routes/users.js

```diff
@@ -7,6 +7,16 @@ const QRCode = require('qrcode')
 
 /* GET users listing. */
 router.get('/', async (req, res, next) => {
+
+  const users = await User.findAll()
+
+  if (req.user.level === User.ADMIN) {
+    return res.json({
+      status: 'ok',
+      result: users
```

Comment:
We don't want to return the raw data from the query because this will, by default, contain sensitive information (specifically multi-factor codes and hashed passwords). It's probably better to use the `toJSON` method I built on it. If you need more than what it returns by default, have an optional admin flag on the function call to determine what to send down.